lib/rules/max-attributes-per-line.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @fileoverview Define the number of attributes allows per line
+ * @author Filipa Lacerda
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Define the number of attributes allows per line',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: null,
+    schema: [{
+      type: 'object',
+      properties: {
+        'singleline': {
+          type: 'number',
+          minimum: 1
+        },
+        'multiline': {
+          type: 'object',
+          properties: {
+            'max': {
+              type: 'number',
+              minimum: 1
+            },
+            'allowFirstLine': {
+              type: 'boolean'
+            }
+          }
+        }
+      }
+    }]
+  },
+
+  create: function (context) {
+    const configuration = context.options[0] || { singleline: 3, multiline: { max: 1, allowFirstLine: false }}
+    const multilineMaximum = configuration.multiline.max
+    const singlelinemMaximum = configuration.singleline
+    const canHaveFirstLine = configuration.multiline.allowFirstLine
+
+    utils.registerTemplateBodyVisitor(context, {
+      'VStartTag' (node) {
+        const numberOfAttributes = node.attributes.length
+
+        if (!numberOfAttributes) return
+
+        if (isSingleLine(node) && numberOfAttributes > singlelinemMaximum) {
+          context.report({
+            node,
+            loc: node.loc,
```

Comment:
I still think better if this rule reports the violated attributes instead of elements which have the violated attributes.
This is consistent with the core rules which have similar functionally.

```js
/*eslint 
    max-statements-per-line: error,
    newline-per-chained-call: error,
    object-property-newline: error,
    array-element-newline: error
 */

aaa(); bbb(); ccc();
test.aaa().bbb().ccc().ddd()
var obj = {aaa:1, bbb:2, ccc:3}
var ary = [1, 2, 3]
```

![image](https://user-images.githubusercontent.com/1937871/28390317-76d78cec-6d14-11e7-8df0-725ca061d0f8.png)